samples/net/sockets/echo_client/sample.yaml

```diff
@@ -70,6 +70,14 @@ tests:
   sample.net.sockets.echo_client.nrf_802154:
     extra_args: EXTRA_CONF_FILE="overlay-802154.conf"
     platform_allow: nrf52840dk/nrf52840
+  sample.net.sockets.echo_client.mcxw71_openthread:
```

Comment:
Can we have a single `sample.net.sockets.echo_client.openthread` entry instead and just list the supported platforms in the `platform_alow:` section? It doesn't really make sense to multiply those build variants given the only difference is the platform.

Reply:
Done, please check.
Thanks

Reply:
Thanks